lib/travis/model/repository/settings.rb

```diff
@@ -111,6 +116,17 @@ def timeout_log_silence
     value == 0 ? nil : value
   end
 
+  def rate_limit
+    super || 0
+  end
+
+  def rate_limit_restriction
+    if rate_limit > 200
```

Comment:
I think we should move the hardcoded max limit to the config. E.g. `Travis.config.settings.maximus.api_builds`. That's defined in `travis/config/defaults.rb`